ingest_operations.py

```diff
@@ -11,6 +11,26 @@
 
 app = Flask(__name__)
 
+ERROR_CODES = {
+    "SUCCESS": 0,
+    "UNAUTHORIZED": 1,
+    "VALIDATION": 2,
+    "COHORTEXISTS": 3,
+    "INTERNAL": 4,
+    "AUTHORIZATIONERR": 5
```

Comment:
is this a typo with double R?

Reply:
Just short form for error. But I can change it